.env.development

```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_GEN3_COMMONS_NAME=gen3
 NEXT_PUBLIC_GEN3_API=https://qa-mickey.planx-pla.net
+NEXT_PUBLIC_GEN3_DOMAIN=https://localhost:3010
```

Comment:
what do we mean with `NEXT_`?

Reply:
sorry that was from my local configs did not mean to include, thanks for checking 
```suggestion
```